PYME/Acquire/ui/tile_panel.py

```diff
@@ -111,7 +111,7 @@ def _launch_viewer(self):
         try:
             requests.get('http://127.0.0.1:8979/set_tile_source?tile_dir=%s' % self.scope.tiler._tiledir)
         except requests.ConnectionError:
-            self._gui_proc = subprocess.Popen('%s -m PYME.tileviewer.tileviewer %s' % (sys.executable, self.scope.tiler._tiledir), shell=True)
```

Comment:
This is not a 1:1 substitution, as `shell=True` implies a bunch of other things in addition to `CREATE_NEW_CONSOLE` (we might run into issues with shell expansion of the command line amoungst other things - specifically you will no longer be able to use paths relative to, e.g. home, but I suspect there might be other issues. More concerningly, it will also causer the tileviewer to fail on non-windows platforms as `subprocess.CREATE_NEW_CONSOLE` is windows only.

Reply:
not sure what you mean by shell expansion of the command line, but this turns out to be a nice little feature to be able to kill/restart the tileviewer if I e.g. brick the GPU on a detection recipe, without having to restart PYMEAcquire or guess on windows which python process it is.

I'll add a try/except attribute error to catch for non-windows users. Will also check for abspath on tile_dir and add cwd if it's not. 

I don't see anything in shell=False that would be a problem for us since we are otherwise not using pipes / ~ / etc.

Reply:
Previously you absolutely could have used a path relative to `~`, but I'm not super attached to doing that. 

If it's not an absolute path, it's probably best to drop it under `PYMEDATADIR` rather than `cwd()` as the working directory is often weird if you are running using a windows shortcut. Will see if I can change to the relevant nameutils call.

I'm still not sure why we're not running into  *"If passing a single string, either shell must be True (see below) or else the string must simply name the program to be executed without specifying any arguments."* though (suspect windows might be doing some limited expansion/parsing under the hood - I've definitely seen breakages related to the single string stuff on *nix.

Reply:
I've done most of the testing on *nix so have either killed as needed with `ps -A | grep tileviewer`, or launched the tileviewer separately in advance. One of the advantages of using `shell=True` is that you end up with the argument in the process name.